exo-go/src/docker_compose/index.go

```diff
@@ -0,0 +1,43 @@
+package dockerCompose
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os/exec"
+)
+
+// RunAllImages runs all the docker images
+func RunAllImages(env []string, cwd string, log func(string)) (*exec.Cmd, *bytes.Buffer, error) {
+	return start(cwd, env, log, "docker-compose", "up")
+}
+
+// KillAllContainers kills all the containers
+func KillAllContainers(env []string, cwd string, log func(string)) (*exec.Cmd, *bytes.Buffer, error) {
+	return start(cwd, env, log, "docker-compose", "down")
+}
+
+func start(dir string, env []string, log func(string), command ...string) (*exec.Cmd, *bytes.Buffer, error) {
```

Comment:
I had to write this function although it is kind of similar to `processHelpers.Start` because I want to be able to pass in environment variables and the log function so that I can log the output as the process is run. Also, I had to use both stdout pipe and a std buffer because I want to store the output so we can wait for online texts on top of doing the logging. I'm not very familiar with Go yet so need suggestions for a better way to do this.